test/unit/handlers/lib/kafka/producer.test.js

```diff
@@ -177,7 +177,7 @@ Test('Producer', producerTest => {
         test.ok(Producer.getProducer('undefined'))
         test.fail('Error not thrown!')
       } catch (e) {
-        test.ok(e.toString() === 'Error: No producer found for topic undefined')
+        test.equal(e.message, 'No producer found for topic undefined')
```

Comment:
Producer.getProducer() throws the following:

throw ErrorHandler.Factory.createInternalServerFSPIOPError(`No producer found for topic ${topicName}`)

so instead of adding:

test.assert(error instanceof Errors.ValidationError)

added:

test.assert(e instanceof FSPIOPError)